src/io/box_filtering.py

```diff
@@ -0,0 +1,28 @@
+"""
+Define same filtering that we apply in:
+"Learning to detect objects on a 1 Megapixel Event Camera" by Etienne Perot et al.
+
+Namely we apply 2 different filters:
+1. skip all boxes before 0.5s (before we assume it is unlikely you have sufficient historic)
+2. filter all boxes whose diagonal <= min_box_diag**2 and whose side <= min_box_side
+
+Copyright: (c) 2019-2020 Prophesee
+"""
+from __future__ import print_function
+import numpy as np
+
+
+def filter_boxes(boxes, skip_ts=int(1e5), min_box_diag=30, min_box_side=10):
```

Comment:
is this at the original scale or downscaled? we maybe need to clarify this

Reply:
right, i think we use min_box_diag=30 in our case, both for imerit and the moorea dataset (60/2 and 20/2). We have not evaluated on Full resolution

Reply:
will add a bit of text in the README